es/cheatsheet-deep-learning.md

```diff
@@ -1,291 +1,291 @@
 **1. Deep Learning cheatsheet**
 
-&#10230;
+&#10230; Hoja de referencia de Deep Learning
```

Comment:
Use "_Aprendizaje Profundo (Deep Learning, en inglés)_". It is a term that is easy to find in Spanish literature.